src/kirby/components/list/list.component.scss

```diff
@@ -3,61 +3,90 @@
 $divider-color: get-color('background-color');
 $background-color-hover: get-color('background-color');
 $shadow: get-elevation(2);
+$item-height: size('xxxl');
 
-:host {
-  ion-list {
-    box-shadow: $shadow;
-    background: transparent;
-    contain: inherit;
-    padding: 0;
+ion-list {
+  box-shadow: $shadow;
+  background: transparent;
+  contain: inherit;
+  padding: 0;
+}
+
+ion-item {
+  min-height: $item-height;
+  background-color: get-color('white');
+  &:not(:last-of-type).has-divider {
+    border-bottom: 1px solid $divider-color;
   }
 
-  ion-item {
-    background-color: get-color('white');
-    &:not(:last-of-type).has-divider {
-      border-bottom: 1px solid $divider-color;
-    }
+  &.selected ::ng-deep * {
+    font-weight: font-weight('bold');
+  }
+  &:focus {
+    $outline-width: 5px;
+    outline-color: var(--highlight-color-focused);
+    outline-style: auto;
+    outline-width: $outline-width;
+    margin-bottom: $outline-width;
+    background-color: $background-color-hover;
+  }
 
-    &.selected ::ng-deep * {
-      font-weight: font-weight('bold');
-    }
-    &:focus {
-      $outline-width: 5px;
-      outline-color: var(--highlight-color-focused);
-      outline-style: auto;
-      outline-width: $outline-width;
-      margin-bottom: $outline-width;
+  &.selectable {
+    cursor: pointer;
+    &:hover {
       background-color: $background-color-hover;
     }
+  }
 
-    &.selectable {
-      cursor: pointer;
-      &:hover {
-        background-color: $background-color-hover;
-      }
-    }
+  $list-colors: ('light'); // add supported list item theme colors here
+  @each $color-name, $color-value in $list-colors {
+     &.#{$color-name} {
+             background-color: #{get-color($color-name)};
```

Comment:
There is some indentation here that is off.

Reply:
Not related to this PR, but fixed :)